contrib/asmap/README.md

```diff
@@ -9,4 +9,73 @@ Example usage:
 python3 asmap-tool.py encode /path/to/input.file /path/to/output.file
 python3 asmap-tool.py decode /path/to/input.file /path/to/output.file
 python3 asmap-tool.py diff /path/to/first.file /path/to/second.file
+python3 asmap-tool.py diff-addrs /path/to/first.file /path/to/second.file addrs.file
+```
+These commands may take a few minutes to run with `python3`,
+depending on the amount of data involved and your machine specs.
+Consider using `pypy3` for a faster run time.
+
+### Encoding and Decoding
+
+ASmap files are somewhat large (~30M) in text form, and should be encoded
+to binary before being used with Bitcoin Core.
```

Comment:
"need to be" instead of "should be" (it's not only because of the size, AFAIK bitcoin core just can't parse text files)

Reply:
[704573e](https://github.com/bitcoin/bitcoin/pull/32110/commits/704573e016ac40e7cae00a84a9ba532a760e043e)